examples/basics/custom_embeddings_sdk.ipynb

```diff
@@ -39,7 +39,7 @@
     {
```

Comment:
<p>Can you remove the line above emeddings</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1582/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Done